### ISSUE TITLE
Updating method of Learning rate

### ISSUE THREAD

**Issue by LB15642219539** [mk-minchul/AdaFace#102]
Hello, I recently studied your Adaface Loss function and your code framework. Is there anything special about the adjustment of Learning rate in your framework? Using lr_ The scheduler. MultiStepLR() method also lacks warmup. Your explanation will help me better understand!
